packages/now-client/src/deployment-status.ts

```diff
@@ -13,19 +13,24 @@ export default async function* checkDeploymentStatus(
   deployment: Deployment,
   token: string,
   version: number | undefined,
-  teamId?: string
+  teamId: string | undefined,
+  debug: Function
 ): AsyncIterableIterator<DeploymentStatus> {
   let deploymentState = deployment;
   let allBuildsCompleted = false;
   const buildsState: { [key: string]: DeploymentBuild } = {};
   let apiDeployments = version === 2 ? API_DEPLOYMENTS : API_DEPLOYMENTS_LEGACY;
 
+  debug(`Using ${version}.0 API for status checks`);
```

Comment:
This message will look strange when `version` is undefined.

Reply:
Technically it shouldn't be undefined since it's set by the main function factory 🤔
I'll add a safeguard just in case

Reply:
If that's true, then you can change the function parameter to be `number` instead of `number | undefined`